core-services/template-deprecation/_allowlist.yaml

```diff
@@ -4183,6 +4183,9 @@ templates:
         pull-ci-openshift-elasticsearch-operator-master-e2e-operator:
           generated: true
           kind: presubmit
+        pull-ci-openshift-elasticsearch-operator-master-e2e-upgrade:
```

Comment:
Please also remove this, it isn't needed anymore now and will allow anyone in your team to approve this PR rather than requiring dptp approval